src/current/cockroachcloud/cloud-api.md

```diff
@@ -43,7 +47,7 @@ Authorization: Bearer {secret_key}
 
 </section>
 
-Where `{secret_key}` is the [secret key string you stored when you created the API key in the Console]({% link cockroachcloud/managing-access.md %}#create-api-keys).
+Where `{secret_key}` is the secret key string you stored when you [created the API key in the Console]({% link cockroachcloud/managing-access.md %}#create-api-keys).
 
 ## Set the API version
 
```

Comment:
This is out of sync with https://github.com/cockroachdb/docs/pull/18918, can you please review that so this PR will benefit from it?